server/src/main/java/org/elasticsearch/rest/action/search/RestSearchAction.java

```diff
@@ -201,7 +197,12 @@ public static void parseSearchRequest(
         checkRestTotalHits(request, searchRequest);
 
         if (searchRequest.pointInTimeBuilder() != null) {
-            preparePointInTime(searchRequest, request, namedWriteableRegistry);
+            if (request.paramAsBoolean("ccs_minimize_roundtrips", false)) {
+                throw new IllegalArgumentException("[ccs_minimize_roundtrips] cannot be used with point in time");
+            }
+            if (searchRequest.indicesOptions().equals(DEFAULT_INDICES_OPTIONS) == false) {
+                throw new IllegalArgumentException("[indicesOptions] cannot be used with point in time");
+            }
```

Comment:
Along the same lines as above, I wonder if we should try and share the validation that we already have in SearchRequest